docs/docs/user/migration-guide/v3-v4.md

```diff
@@ -0,0 +1,32 @@
+# Migrate from V2 to V3
+
+## [Distribute defaults across multiple files](https://github.com/bakdata/kpops/pull/438/)
+
+The default files are now read from where the pipeline path is defined. The `--defaults` flag is removed. Imagine the following folder structure:
```

Comment:
In the first sentence, I'd add a brief explanation that multiple defaults files are now possible, before going into detail